### PR TITLE
add referring provider to db en route to encounter form

### PR DESCRIPTION
**Pull request by stephenwaite**
#### Short description of what this resolves:
eventually add referring provider to the encounter form following a forum [thread])https://community.open-emr.org/t/referring-provider-per-encounter/17765?u=stephenwaite)

### CODE REVIEW

**Comment by adunsulag** [comment]
If we add referring provider we probably will want to add this to the FHIR interface as a participant with a participant type of REF.  Not suggesting you do that here, but maybe we open up on issue to track this so we can get it done eventually?

**Comment by bradymiller** [comment]
tenor99

**Comment by stephenwaite** [comment]
testing well

**Comment by bradymiller** [comment]
looks great! looks like somebody beat you to the delete fix :)

2DV

**Review by bradymiller — APPROVED** [review]

File: interface/patient_file/summary/demographics.php [hunk 1]
```diff
@@ -1619,8 +1619,8 @@ function setMyPatient() {
                         echo $twig->getTwig()->render('patient/partials/delete.html.twig', [
                             'isAdmin' => AclMain::aclCheckCore('admin', 'super'),
                             'allowPatientDelete' => $GLOBALS['allow_pat_delete'],
-                            'pid' => $pid,
-                            'csrf' =>  CsrfUtils::collectCsrfToken(),
+                            'csrf' => CsrfUtils::collectCsrfToken(),
```
Comment (sjpadgett) [inline interface/patient_file/summary/demographics.php:1622]: nan na na nan nan! too slow:)

**Pull request merged**